setup/3. Importing.md

```diff
@@ -0,0 +1,19 @@
+# Importing your Project
+
+## Importing into IntelliJ
+
+If you followed the `Cloning from your IDE` step in the `Git and Github` tutorial, you can skip ahead to `Launching Minecraft`.
+
+Otherwise, if you downloaded the ZIP or cloned your project with git manually, open IntelliJ.
+A window will open with an option to `Open or Import`. Select this and click on the folder where your project is in. 
+The folder you click on should have files like `build.gradle` and a folder named `src` in it. Do not select the folder above your project 
```

Comment:
> Do not select the folder above your project 

You should clarify this a bit - I thought you meant a folder next to the project.